mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -738,6 +738,8 @@
     <string name="pwa_site_controls_title_private">%1$s (Private Mode)</string>
     <!-- Button in the current tab tray header in multiselect mode. Saved the selected tabs to a collection when pressed. -->
     <string name="tab_tray_save_to_collection">Save</string>
+    <!-- Text for normal tabs header title. -->
```

Comment:
@jonalmeida I have a hard time understanding how the comment relates to its string "Other", thinking localizers might so too. Could you update it, giving more context?

Reply:
Yeah, sorry about that. How about "Title text for the normal tabs header in the tabs tray which are not part of any tab grouping" @rocketsroger ?

Reply:
Sounds good to me

Reply:
Fix is up https://github.com/mozilla-mobile/fenix/pull/21365.